test/oauth-authorization-test.js

```diff
@@ -335,24 +351,38 @@ suite.addBatch({
                         requestToken(cl, this);
                     },
                     function(err, rt) {
+                        var self = this;
+
                         if (err) throw err;
-                        br = new Browser({runScripts: false});
-                        br.visit("http://localhost:4815/oauth/authorize?oauth_token=" + rt.token, this);
+                        br = new Browser({runScripts: false}); // faster without scripts...
+                        Step(
+                            function() {
+                                br.visit("http://localhost:4815/oauth/authorize?oauth_token=" + rt.token, this);
+                            },
+                            function() {
+                                self(!br.success, br);
+                            }
+                        );
                     },
                     function(err, br) {
+                        var self = this;
```

Comment:
Please can you un-subscribe my email address shiju.varghese@dekhdekh.com <mailto:shiju.varghese@dekhdekh.com> so that I do not receive emails like the one below?

Many thanks.

> On 10 Jan 2016, at 10:42, Menno Vossen <notifications@github.com> wrote:
> 
> In test/oauth-authorization-test.js <https://github.com/e14n/pump.io/pull/1136#discussion_r49274288>:
> 
> >                      },
> >                      function(err, br) {
> > +                        var self = this;
> In the last function as callback
> br.visit("bla",this)... calls next function in Step. self(!br.success, br); calls outer Step
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/e14n/pump.io/pull/1136/files#r49274288>.
>